Source/Particles/ParticleBuffer.H

```diff
@@ -0,0 +1,33 @@
+/* Copyright 2021 Andrew Myers
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef PARTICLEBUFFER_H_
+#define PARTICLEBUFFER_H_
+
+#include "Particles/MultiParticleContainer.H"
+#include "WarpX.H"
+
+#include <AMReX_AmrParticles.H>
+
+namespace ParticleBuffer {
+    template <template<class> class Allocator>
+    using BufferType = amrex::AmrParticleContainer<0, 0, PIdx::nattribs, 0, Allocator>;
```

Comment:
We'll reuse this type as well for BTD as we just realized (via @RevathiJambunathan :) )